docs/decisions/0008-compatibility-scheme.rst

```diff
@@ -0,0 +1,58 @@
+0008: Compatibility scheme with the current system
+###################################################
+
+Status
+******
+
+**Draft** *2025-09-29*
+
+Context
+*******
+
+Open edX has its authorization system described in the `OEP-66`_, but due to its limitations, the community wanted to explore a more appropriate option for managing authorization on the platform. To mitigate the possible risk associated with completely overhauling a core system like authorization, our primary strategy is to implement a staging or phased migration plan. This approach enables us to limit the blast radius to test components in a controlled environment, apply lessons learned, and ensure business continuity, thereby giving users time to adapt.
+
+Decision
+********
+
+* The new authorization will coexist with the previous one until we migrate the entire system.
+* We will start migrating the current library permissions and roles to the new authorization system.
+    * For the MVP, we will maintain the current functionality using the new architecture.
+
+Consequences
+************
+
+Migration Strategy for Libraries
+=================================
+
+* Develop a migration script to transform the existing explicit role assignments to the new authorization model, without modifying the previous table.
```

Comment:
Can we with the current data migrate this entirely to our new policy? Would it be a 1:1 mapping? I think this question applies for the rest of the modifications. I understand each step of the migration is different and maybe some map 1:1 and others don't, but including that initial study as a mandatory step as part of the consequences makes total sense to me.